index.js

```diff
@@ -19,16 +18,7 @@ function sizeof(object) {
       else {
         var bytes = 0;
         _.forOwn(object, function (value, key) {
```

Comment:
if ES6 will be used to store keys and values, you may iterate not only by own properties. Functions in prototype also consume memory